src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/nowhitespaceafter/InputNoWhitespaceAfterFormerNpe.java

```diff
@@ -0,0 +1,8 @@
+package com.puppycrawl.tools.checkstyle.checks.whitespace.nowhitespaceafter;
+
+public class InputNoWhitespaceAfterFormerNpe
```

Comment:
This input file looks like it is only used by this test.
We expect alone files to be moved, not copied, to their new destination.
Please confirm for this file, and fix including any other files for this test.